### PR TITLE
v1.0 to v1.2 Update: UserGuide, DeveloperGuide

### PR DESCRIPTION
**Pull request by Sheikh-Umar**
Updated all references to Address Book ("address book", "CRM") to "CRM Book"
Added a user story
Edited v1.0 to v1.2 documentation

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -1070,7 +1063,9 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |salesperson |see my calendar |schedule meetings without conflicting
 
-|`* *` |salesperson |schedule a meeting using Google Hangouts |meet my leads online more efficiently
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: why did you remove this

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -1070,7 +1063,9 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+|`*` |salesperson |see which Leads and Contacts are dealing with the same type of product|recommend potential sales partners to Leads who are seeking sales experience
+
+|`*` |salesperson |schedule a meeting using zoom |schedule meetings easily and with a link other salespeople can use
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: i removed this in a prev PR cuz it's not feasible as it costs money. Pls remove

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (davidten) [inline docs/DeveloperGuide.adoc]: no, that's not what a salesperson does

Comment (Sheikh-Umar) [reply docs/DeveloperGuide.adoc]: Accidentally removed it. Sorry.

Comment (Sheikh-Umar) [reply docs/DeveloperGuide.adoc]: I've made the necessary changes.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -1070,22 +1063,24 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |salesperson |see my calendar |schedule meetings without conflicting
 
-|`* *` |salesperson |schedule a meeting using Google Hangouts |meet my leads online more efficiently
+|`*` |salesperson |schedule a meeting using zoom |schedule meetings easily and with a link other salespeople can use
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: please remove the zoom use case

Comment (Sheikh-Umar) [reply docs/DeveloperGuide.adoc]: I have removed it.
